test/mocha/10-api.js

```diff
@@ -32,14 +32,10 @@ describe('bedrock-account-http', function bedrockAccountHttp() {
     passportStub.restore();
   });
   describe('get /', function getIndex() {
-    it('should return 404 with no email', async function worksGreat() {
+    it('should return 400 with no email', async function worksGreat() {
       const result = await api.get('/');
-      const {status, data} = result;
-      should.equal(status, 404);
-      data.should.be.an('object');
-      const {message, type} = data;
-      message.should.match(/account does not exist/i);
-      type.should.match(/NotFoundError/i);
+      const {status} = result;
+      should.equal(status, 400);
```

Comment:
Validation middleware does return an error in response.body, so you *could* make some light asserations about that.